domains/authorization/Proxy.IntegrationTests/ProxyBaseIntegrationTests.cs

```diff
@@ -11,21 +9,21 @@ namespace Proxy.IntegrationTests;
 
 public class ProxyBaseIntegrationTests(ProxyIntegrationTestFixture fixture) : IClassFixture<ProxyIntegrationTestFixture>
 {
-    private HttpClient CreateClientWithOrgIds(List<string> orgIds)
+    private HttpClient CreateClientWithOrgIds(Guid orgId, List<string> orgIds)
     {
-        return fixture.Factory.CreateAuthenticatedClient(orgIds: orgIds);
+        return fixture.Factory.CreateAuthenticatedClient(orgId: orgId.ToString(), orgIds: orgIds);
```

Comment:
Skal den string transformeres? kan metoden refaktureres til at tage imod OrgId typen direkte?

Reply:
The _CreateAuthenticatedClient_ method uses primitive types in other parameters, so I chose to keep it that way.